### PR TITLE
Add packages to $PackageExclusions which do not build properly in Docs CI

### PR DESCRIPTION
**Pull request by danieljurek**

### CODE REVIEW

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -288,6 +288,15 @@ $PackageExclusions = @{
   'azure-mgmt-signalr' = 'Unsupported doc directives https://github.com/Azure/azure-sdk-for-python/issues/18085';
   'azure-mgmt-mixedreality' = 'Missing version info https://github.com/Azure/azure-sdk-for-python/issues/18457';
   'azure-mgmt-network' = 'Manual process used to build';
+
+  'azure-mgmt-compute' = 'Latest package requires Python >= 3.7 and this breaks docs build. https://github.com/Azure/azure-sdk-for-python/issues/22492';
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:292]: This is going to soon be almost all python packages do we have a plan to get the docs team to support this?

Comment (scbedd) [reply eng/scripts/Language-Settings.ps1:292]: @danieljurek already has a PR out with them.

**Review by scbedd — APPROVED** [review]

**Review by weshaggard — APPROVED** [review]

**Comment by scbedd** [comment]
@danieljurek ignore the failures or requeue the builds. The conda failures are unrelated to this PR and they've been disabled temporarily in `main`.

**Pull request merged**